server/src/quic/listener.rs

```diff
@@ -48,7 +48,7 @@ pub fn start(endpoint: Endpoint, system: SharedSystem) {
 async fn handle_connection(
     incoming_connection: quinn::Connecting,
     system: SharedSystem,
-) -> Result<(), ServerError> {
+) -> Result<(), ServerConnectionError> {
```

Comment:
I think we can drop the `Server` prefix from the error name `ConnectionError` should be sufficient and we will use the top level crate name to distinct between server and sdk errors (crates - sdk / iggy )

Reply:
done